packages/mui/src/pickers-common/default-mui-picker-localization.tsx

```diff
@@ -7,6 +7,8 @@ import { Locale } from '../constant-types';
 
 interface Props {
   locale: Locale;
+  lang?: Locale;
+  localeTexts?: Partial<PickersLocaleText>;
```

Comment:
remove

Reply:
I differentiated between lang and locale.
locale determines calendar system and lang determines the language used for locale text and other labels!